liquidcore/home/templates/homepage.html

```diff
@@ -1,8 +1,10 @@
 {% extends "base.html" %}
 
+{% load staticfiles %}
+
 {% block content %}
 <h1 class="shady">Liquid Investigations</h1>
-<h1>{{ liquid_title }}</h1>
```

Comment:
Don't remove the title, it's the name of the deployment, quite important for people to know where they are. If you want to avoid having too many "liquid investigations" displayed on the example homepage then change the title in `examples/liquid.ini`.